src/routes/index.ts

```diff
@@ -254,8 +254,6 @@ router.post('/cmd/sync', async (req, res) => {
   }
 });
 
-
-
 router.get('/sensordata/:since', async (req: Request, res: Response) => {
```

Comment:
This endpoint is no longer needed, but I don't want to break sensor telemetry on the old mobile app firmware.  What should we do with this?